api/ReadableStream.json

```diff
@@ -52,7 +52,7 @@
           "spec_url": "https://streams.spec.whatwg.org/#ref-for-rs-constructor⑤",
           "support": {
             "chrome": {
-              "version_added": "43"
+              "version_added": "52"
```

Comment:
How did things behave in Chrome 43-51? Was the interface really exposed, but it didn't work? I suspect more changes are needed to make the data consistent around this.

Reply:
That seems to be the case, yes.  In Chrome 51, `new ReadableStream()` simply returns `Illegal constructor`.

Reply:
I see, thanks for checking!